lib/build.js

```diff
@@ -61,6 +61,13 @@ module.exports = {
       default: true,
       order: 5
     },
+    showBuildNotifications: {
```

Comment:
Why do you want to add notifications? The build panel should be notification enough, no?